modules/seqexec/server/src/test/scala/seqexec/server/CleanConfigSpec.scala

```diff
@@ -30,7 +30,7 @@ class CleanConfigSpec extends AnyFlatSpec with Matchers {
     )
   )
 
-  "CleanConfig" must "fail construction if given a set of overrides with conflicting types" in {
+   ignore must "fail construction if given a set of overrides with conflicting types" in {
```

Comment:
I removed the test after I had to remove the assertion in `CleanConfig`. Some day we will have a properly typed configuration.